HowTos/Cloud_Networking_Ref_Des.rst

```diff
@@ -55,41 +55,14 @@ VPC ID and its region for the VPC ID field and region in each step.
 
 1. Launch a gateway with VPN capability in VPC 172.31.0.0/16.
 
-   a. Go to the Gateway menu and create. Make sure:
+   a. Go to the Gateway menu and create a gateway. 
 
-   #. At the Gateway Name field, give it a distinct and convenient name. For
+   b. At the Gateway Name field, give it a distinct and convenient name. For
       example, mgmt-vpn-1
 
-   #. Select VPN Access.
-
-   #. Use the default VPN CIDR Block.
+   c. Select VPN Access. Selecting features in the Advanced Options section is not required. Selecting "Split Tunnel Mode" is not required.
 
-   #. Select Split Tunnel Mode.
```

Comment:
We need to retain this info.. the options are still avaliable under advanced

Reply:
the user does not need to specify, and it can be changed later. We want to simplify at this point.